integr8ly/cloud-resource-operator
# MGDAPI-4539 add gcp provider skeleton and creds req

## Overview

Jira: https://issues.redhat.com/browse/MGDAPI-4539

## Verification

- Checkout this [branch of Delorean](https://github.com/integr8ly/delorean/pull/283) and create a GCP CCS cluster:
```
make -f make/ocm.mk ocm/cluster.json BYOC=true CLOUD_PROVIDER=gcp
```
- After the cluster installation completes, set up the cluster, using CRO:
```
make cluster/prepare
```
- Run CRO locally:
```
make run
```
- In another tab, create new blobstorage/postgres/redis instance:
```
make cluster/seed/blobstorage PROVIDER=gcp
```
- With the current implementation, the only thing that we should expect is to have no errors reconciling `CredetialsRequest`. No cloud resources are expected to be created at this point because there's no implementation for that functionality yet.
